### PR TITLE
docs: provide updates to PyCon- and SciPy-related blog posts

### PR DESCRIPTION
**Pull request by rich-iannone**
This updates two blogposts focused on PyCon 2024 and SciPy 2024 events. Some dead/non-working links are updated here. Additional links to new resources (recorded videos) are provided here.

Fixes: https://github.com/posit-dev/great-tables/issues/180

### CODE REVIEW

File: docs/blog/pycon-2024-great-tables-are-possible/index.qmd [hunk 1]
```diff
@@ -79,9 +79,10 @@ We’ll cover the following:
 
 Check out these resources to learn more about the wild and beautiful life of display tables:
 
-* [Great Tables example gallery](/docs/examples)
```
Comment (machow) [inline docs/blog/pycon-2024-great-tables-are-possible/index.qmd:82]: shoot some of these broken links were my bad -- but it looks like the fixes may also be broken 😓.

File: docs/blog/pycon-2024-great-tables-are-possible/index.qmd [hunk 2]
```diff
@@ -79,9 +79,10 @@ We’ll cover the following:
...
-* [The Design Philosophy of Great Tables (blog post)](http://localhost:6877/blog/design-philosophy/)
+* [Great Tables example gallery](../examples/index.html)
+* [The Design Philosophy of Great Tables (blog post)](blog/design-philosophy/)
```
Comment (machow) [inline docs/blog/pycon-2024-great-tables-are-possible/index.qmd:83]:
```suggestion
* [The Design Philosophy of Great Tables (blog post)](/blog/design-philosophy)
```

File: docs/blog/pycon-2024-great-tables-are-possible/index.qmd [hunk 3]
```diff
@@ -79,9 +79,10 @@ We’ll cover the following:
 
 Check out these resources to learn more about the wild and beautiful life of display tables:
 
-* [Great Tables example gallery](/docs/examples)
-* [The Design Philosophy of Great Tables (blog post)](http://localhost:6877/blog/design-philosophy/)
+* [Great Tables example gallery](../examples/index.html)
```
Comment (machow) [inline docs/blog/pycon-2024-great-tables-are-possible/index.qmd:82]:
```suggestion
* [Great Tables example gallery](/examples)
```

**Review by machow — CHANGES_REQUESTED** [review]
I think the URL tweaks I suggest should fix the broken ones (which I introduced in the original blog post 😭 )

**Comment by rich-iannone** [comment]
I didn't realize the suggestions were up here so I pretty much independently fixed up the links (and tested locally a few times while iterating). That said, things are looking functional now!

**Pull request merged**